.github/workflows/foreman_plugin.yml

```
@@ -52,7 +52,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       postgres:
-        image: 'postgres:${{ matrix.postgresql }}'
+        image: 'ghcr.io/evgeni/postgresql-evr:${{ matrix.postgresql }}'
```

Can we make this an input? Something like:
```suggestion
        image: '${{ inputs.postgresql_image_name }}:${{ matrix.postgresql }}'
```

---

like that?